src/v3/src/util/passwordUtils.test.ts

```diff
@@ -278,6 +278,25 @@ describe('PasswordUtils Tests', () => {
       .toEqual(true);
   });
 
+  it('should validate maxConsecutiveRepeatingCharacters', () => {
+    expect(validatePassword('somepass', userInfo, { complexity: { maxConsecutiveRepeatingCharacters: 3 } })?.maxConsecutiveRepeatingCharacters)
+      .toEqual(true);
+    expect(validatePassword('AaAaAaAaAaaAAaaAA', userInfo, { complexity: { maxConsecutiveRepeatingCharacters: 3 } })?.maxConsecutiveRepeatingCharacters)
+      .toEqual(true);
+    expect(validatePassword('tessst', userInfo, { complexity: { maxConsecutiveRepeatingCharacters: 3 } })?.maxConsecutiveRepeatingCharacters)
```

Comment:
Can you clarify what should fail the validator? This test case shows 3 `s` characters in a row but it's passing the validator. I noticed that the regex below is written in a way that it will consume the first instance of the character, so it only matches when `limit + 1` characters are consecutive.

Reply:
if the value of the password requirement is set to 3, it means we allow a maximum of that many consecutive repeating characters and if we only have 3, it should pass the validation - 4 would fail it in this case

Reply:
doh. Yes that makes sense. Thanks!